templates/corosync.conf.j2

```diff
@@ -1,4 +1,9 @@
-{% set _pacemaker_corosync_bind_addr = hostvars[inventory_hostname]['ansible_' + pacemaker_corosync_ring_interface | replace('-', '_')].ipv4.address %}
+{%  if inventory_hostname in groups[pacemaker_corosync_group] %}
```

Comment:
However this change does not make much sense to me, and I guess I need to revert it. Why you should be obliged to use eth1 interface for pacemaker_corosync_group? I think that user should leverage either group_vars or another variable should be added to control interface for different groups.